examples/sensor-data-java/README.md

```diff
@@ -130,7 +130,7 @@ $ kubectl apply -f claim.yaml
 * Deploy the app.
 
 ```
-$ kubectl cloudflow  deploy -u oauth2accesstoken --volume-mount filter.configuration=claim1   eu.gcr.io/<projectID>/sensor-data-java:9-bbdec44-dirty  -p "$(gcloud auth print-access-token)" valid-logger.log-level=info valid-logger.msg-prefix=valid
+$ kubectl cloudflow  deploy -u oauth2accesstoken --volume-mount filter.configuration=claim1 eu.gcr.io/<projectID>/sensor-data-java:9-bbdec44-dirty -p "$(gcloud auth print-access-token)" valid-logger.log-level=info valid-logger.msg-prefix=valid
```

Comment:
Good to catch this redundant space, in general we would like to avoid fixes irrelevant to the PR.